xml/System.Net.Sockets/Socket.xml

```diff
@@ -1,4 +1,4 @@
-<Type Name="Socket" FullName="System.Net.Sockets.Socket">
+﻿<Type Name="Socket" FullName="System.Net.Sockets.Socket">
```

Comment:
There's an attribute in this file that's causing a build warning on the DuplicateAndClose member. It's called `MonoLimitationAttribute`.

Reply:
Good catch. I'll take a look.